server/tools/peertube-remote.ts

```diff
@@ -0,0 +1,104 @@
+import * as program from 'commander'
+import * as inquirer from 'inquirer'
+const Table = require('cli-table')
+const cfg = require('application-config')('PeerTube/CLI')
+const netrc = require('netrc-parser').default
+let settings = {
+  remotes: []
+}
+cfg.read((err, data) => {
+  if (err) console.error(err)
+  if (data.remotes) settings.remotes = data.remotes
+})
+
+const setPassword = (url, username, password) => {
+  netrc.loadSync()
+  netrc.machines[url] = { login: username, password }
+  netrc.save()
+}
+
+program
+  .usage('[command] [options]')
+
+program
+  .command('add')
+  .description('registers your accounts on remote instances for easier use')
+  .option('-u, --url <url>', 'Server url')
+  .option('-U, --username <username>', 'Username')
+  .option('-p, --password <token>', 'Password')
+  .action(options => {
+    if (!options.url || !options.username) {
+      if (!options.url) console.error('--url field is required.')
+      if (!options.username) console.error('--username field is required.')
+      process.exit(-1)
+    }
+    settings.remotes.push(options.url)
+    cfg.write(settings, function (err) {
+      if (err) console.error(err)
+    })
+
+    if (!options.password) {
+      inquirer
```

Comment:
We already use `prompt` for password prompt (see `import-videos.ts`)

Reply:
:+1: